pkg/decoder/tagxl/v1/port152.go

```diff
@@ -7,16 +7,39 @@ import (
 )
 
 type Port152Payload struct {
-	NewRotationState  uint8   `json:"newRotationState"`
-	OldRotationState  uint8   `json:"oldRotationState"`
-	Timestamp         uint32  `json:"timestamp"`
-	NumberOfRotations float64 `json:"numberOfRotations"`
-	ElapsedSeconds    uint32  `json:"elapsedSeconds"`
+	Version           uint8     `json:"version"`
+	SequenceNumber    uint8     `json:"sequenceNumber"`
+	NewRotationState  uint8     `json:"newRotationState"`
+	OldRotationState  uint8     `json:"oldRotationState"`
+	Timestamp         time.Time `json:"timestamp"`
+	NumberOfRotations float64   `json:"numberOfRotations"`
+	ElapsedSeconds    uint32    `json:"elapsedSeconds"`
```

Comment:
Can we maybe introduce some validation? Perhaps the number of rotations is a candidate or elapsed seconds perhaps?